libs/cosmos-sdk/store/rootmulti/rootmulti_store.go

```diff
@@ -468,7 +468,7 @@ func (rs *Store) loadVersion(ver int64, upgrades *types.StoreUpgrades) error {
 	if len(rs.pruneHeights) > maxPruneHeightsLength {
 		return fmt.Errorf("Pruned heights length <%d> exceeds <%d>, "+
 			"need to prune them with command "+
-			"<exchaind data prune-compact all --home your_exchaind_home_directory> before running exchaind",
+			"<exchaind data prune-compact clear-prune-heights --home your_exchaind_home_directory> before running exchaind",
```

Comment:
If we only clear the prune heights. how can we handle the corresponding version?